kobo-uncaged/kunc/kunc.go

```diff
@@ -0,0 +1,296 @@
+// Copyright 2019 Sherman Perry
+
+// This file is part of Kobo UNCaGED.
+
+// Kobo UNCaGED is free software: you can redistribute it and/or modify
+// it under the terms of the Affero GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Kobo UNCaGED is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+
+// You should have received a copy of the GNU Affero General Public License
+// along with Kobo UNCaGED.  If not, see <https://www.gnu.org/licenses/>.
+
+package kunc
+
+import (
+	"fmt"
+	"image"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+	"syscall"
+	"time"
+
+	"github.com/mitchellh/mapstructure"
+	"github.com/pkg/errors"
+	"github.com/shermp/Kobo-UNCaGED/kobo-uncaged/device"
+	"github.com/shermp/Kobo-UNCaGED/kobo-uncaged/kuprint"
+	"github.com/shermp/Kobo-UNCaGED/kobo-uncaged/util"
+	"github.com/shermp/UNCaGED/uc"
+)
+
+type koboUncaged struct {
+	k *device.Kobo
+}
+
+// New initialises the koboUncaged object that will be passed to UNCaGED
+func New(kobo *device.Kobo) *koboUncaged {
+	ku := &koboUncaged{}
+	ku.k = kobo
+	return ku
```

Comment:
Change this into `&koboUncaged{kobo}`, as the long version doesn't really add anything in terms of readability with only one one-letter field.

Reply:
Thanks for catching that